src/main/java/ch/uzh/ifi/hase/soprafs22/service/GameService.java

```diff
@@ -247,7 +247,7 @@ public Game getGameFromRandomUser(User user) {
     return somePage.getContent().get(0);
   }
 
-
+//TODO: saveGame has no usage.. can it be deleted?
     public void saveGame(Game activeGame) {
       gameRepository.saveAndFlush(activeGame);
     }
```

Comment:
strange, that no usage. if no usage: sure. where do we saveandflush game?

Reply:
in createGame. you can delete this!

Reply:
done